drivers/ethernet/phy/phy_mii.c

```diff
@@ -49,6 +49,10 @@ static inline int reg_read(const struct device *dev, uint16_t reg_addr,
 {
 	const struct phy_mii_dev_config *const cfg = dev->config;
 
+	/* if there is no mdio (fixed-link) it is not supported to read */
+	if (!cfg->mdio) {
```

Comment:
Make the `NULL` check explicit.

```suggestion
	if (cfg->mdio == NULL) {
```